dask/array/tests/test_overlap.py

```diff
@@ -447,6 +447,20 @@ def func(*args):
     assert all(x.compute() == size_per_slice)
 
 
+def test_map_overlap_trim_using_drop_axis_and_different_depths():
+    x = da.ones((5, 10), dtype=float)
```

Comment:
I'd like to see some non-2D examples in the tests, as well

Reply:
okay. I should also add a non-uniform boundary